packages/flutter_driver/lib/src/driver.dart

```diff
@@ -15,27 +15,27 @@ import 'health.dart';
 import 'message.dart';
 import 'timeline.dart';
 
-enum TracingCategory {
+enum TimelineStream {
   all, api, compiler, dart, debugger, embedder, gc, isolate, vm
 }
 
-const List<TracingCategory> _defaultCategories = const <TracingCategory>[TracingCategory.all];
+const List<TimelineStream> _defaultStreams = const <TimelineStream>[TimelineStream.all];
 
 // See https://github.com/dart-lang/sdk/blob/master/runtime/vm/timeline.cc#L32
-String _tracingCategoriesToString(List<TracingCategory> categories) {
-  final String contents = categories.map((TracingCategory category) {
-    switch(category) {
-      case TracingCategory.all: return 'all';
-      case TracingCategory.api: return 'API';
-      case TracingCategory.compiler: return 'Compiler';
-      case TracingCategory.dart: return 'Dart';
-      case TracingCategory.debugger: return 'Debugger';
-      case TracingCategory.embedder: return 'Embedder';
-      case TracingCategory.gc: return 'GC';
-      case TracingCategory.isolate: return 'Isolate';
-      case TracingCategory.vm: return 'VM';
+String _timelineStreamsToString(List<TimelineStream> streams) {
+  final String contents = streams.map((TimelineStream stream) {
+    switch(stream) {
```

Comment:
Less the constructor privatization stuff I guess that ends up being about the same amount of mess.

```
class TimelineStream {
  const TimelineStream(this.id);
  final String id;

  static const all = const TimelineStream('all');
  static const api = const TimelineStream('API');
  // ...
}

String _timelineStreamsToString(List<TimelineStream> streams) {
  final String contents = streams.map((TimelineStream stream) => stream.toString()).join(', ');
  return '[$contents]';
}
```
I'm going to leave things as they area since both approaches seem about the same amount of OK.